docs/src/components/App.js

```diff
@@ -14,15 +15,20 @@ const NavigationWithRouter = withRouter(Navigation);
 export default function App(props: Props) {
   const { children } = props;
   return (
-    <Box mdDisplay="flex" direction="row" minHeight="100vh">
-      <Column span={12} mdSpan={2}>
-        <Box padding={4}>
+    <Box minHeight="100vh">
+      <Header />
+
+      <Box smDisplay="flex" direction="row">
+        <Column span={12} smSpan={2}>
           <NavigationWithRouter />
-        </Box>
-      </Column>
-      <Column span={12} mdSpan={10}>
-        <Box padding={4}>{children}</Box>
-      </Column>
+        </Column>
+        <Divider />
+        <Column span={12} smSpan={8}>
```

Comment:
@chrislloyd How'd you feel if we changed this to `smSpan={10} mdSpan={8}`? The docs are looking pretty weird between the `sm` and `md` breakpoints

Before:
![image](https://user-images.githubusercontent.com/127199/39675972-3e589ea0-5117-11e8-874a-6359411c49d8.png)

After:
![image](https://user-images.githubusercontent.com/127199/39675976-471dc9b6-5117-11e8-9b16-67802cef2038.png)